controller/authController.js

```diff
@@ -0,0 +1,56 @@
+const pasteBinUser = require('../models/User');
```

Comment:
`User` not `'PasteBinUser`

Are there any other types of users in your app?

Reply:
I used pasteBinUser because , of the register schema is different and i was having an errors. i will have to drop the database which i do not want to drop.

Reply:
In order to seperate it from other user schema in cloud database

Reply:
Did you not create a new dB for this project?

Reply:
Yes , i did